### DISCUSSION TITLE
Tuf a17 (2021) battery charge limit doesn't working?

### DISCUSSION THREAD

**Discussion by parkdoyoon** [seerge/g-helper#2412] [Q&A]
I can get the battery charge limit working, I even deleted Myasus to prevent the corruption and also tried with all the Asus service off.

**Comment by seerge** [comment]
Please check [FAQ](https://github.com/seerge/g-helper/wiki/FAQ#battery-charge-limiter-is-not-working) , your questions are already answered there :)
